dspace-api/src/main/java/org/dspace/app/orcid/service/OrcidQueueService.java

```diff
@@ -0,0 +1,242 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.orcid.service;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.UUID;
+
+import org.dspace.app.orcid.OrcidQueue;
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Item;
+import org.dspace.core.Context;
+
+/**
+ * Service that handles ORCID queue records.
+ *
+ * @author Luca Giamminonni (luca.giamminonni at 4science.it)
+ *
+ */
+public interface OrcidQueueService {
+
+    /**
+     * Create an OrcidQueue record with the given owner and entity. The type of
+     * operation is calculated based on whether or not the given entity was already
+     * pushed to the ORCID registry.
+     *
+     * @param  context      DSpace context object
+     * @param  owner        the owner item
+     * @param  entity       the entity item
+     * @return              the stored record
+     * @throws SQLException if an SQL error occurs
+     */
+    public OrcidQueue create(Context context, Item owner, Item entity) throws SQLException;
```

Comment:
@LucaGiamminonni : I'd recommend we document that in the JavaDocs then.  e.g. something like this:
```
@param owner  The Researcher Profile (Item) which "owns" the entity.
@param entity  The entity to sync.
```

Basically, the current JavaDocs here are confusing as there's no clarity what an "owner item" is and how it can "own" an Entity.   It makes sense though once you describe that this is talking about the linkage between the Researcher Profile (Person Entity) and its related/linked Entities.  So, improving the JavaDocs would make this clear.